tests/dragonfly/http_conf_test.py

```diff
@@ -75,3 +75,32 @@ async def test_password_on_admin(df_factory):
         async with aiohttp.ClientSession(auth=aiohttp.BasicAuth("default", "XXX")) as session:
             resp = await session.get(f"http://localhost:{server.admin_port}/")
             assert resp.status == 200
+            
+async def test_no_password_on_http_api(df_factory):
+    with df_factory.create(
+        port=1112,
+    ) as server:
+        async with aiohttp.ClientSession(auth=aiohttp.BasicAuth("default", "XXX")) as session:
+            resp = await session.post(f"http://localhost:{server.port}/api", json={["ping"]})
```

Comment:
you send an invalid json. the right expression is `json=["ping"]`

Reply:
mb :D seems like i forgot python, haven't touched it for years.